### ISSUE TITLE
Work with proxy with rotaion ip

### ISSUE THREAD

**Issue by udjfvnsaoepna** [Arriven/db1000n#349]
I have mobile ru proxy with rotation ip every 1 min
`http://login:password@proxyip:port`

I try 
`db1000n.exe -proxy http://login:password@proxyip:port`

But program show my real ip:
```
Checking IP address, attempt #0
countrychecker.go:97: Current country: United States (my real ip)
```

How set proxy correctly ?

**Comment by arfgdev** [comment]
It is a bit tricky task. I am not sure if it is a good idea check the country if we have the list of 10000 proxies for example.

**Comment by Arriven** [comment]
fixed in 0.8.14, see [comment](https://github.com/Arriven/db1000n/issues/424#issuecomment-1082907673) for some additional notes
